libs/react-helpers/src/lib/links/links.ts

```diff
@@ -0,0 +1,25 @@
+/** createDocsLinks returns external documentation links that are specific to an environment
+ * @param {string} docsUrl - the documentation URL for the environment ex. https://docs.vega.xyz/testnet or https://docs.vega.xyz/mainnet (NX_VEGA_DOCS_URL)
+ */
+
+export const createDocsLinks = (docsUrl = 'https://docs.vega.xyz/mainnet') => ({
```

Comment:
Chore (non-blocking): It'd be nice to have the url for the testnet docs in code too. Perhaps docsUrl could be a union type of the two possible urls

Reply:
this has been updated too.

Reply:
I meant something like this:
```suggestion
export const createDocsLinks = (docsUrl: 'https://docs.vega.xyz/mainnet' | 'https://docs.vega.xyz/testnet' = 'https://docs.vega.xyz/mainnet') => ({
```

Reply:
this is how I did it in the initial version. in this review, @mattrussell36 asked me to not make this parameter default and optional because it would be harder to maintain. I can restrict it to these two hardcoded URLs but do we really want to do that? how much does it help? are we going to have docs for more environments, would the URL change?

Reply:
hm, I agree with restricting the URL to be either `'https://docs.vega.xyz/mainnet' | 'https://docs.vega.xyz/testnet'` but let's not pass a default, as @mattrussell36 suggested

Reply:
I updated in last commit